src/styles/typography.js

```diff
@@ -19,19 +19,20 @@ export default function createTypography() {
     ...ifSmallDevice(14, 16),
     fontWeight: WEIGHTS[weight] || WEIGHTS.regular,
   });
+  const disallowExtrabold = weight => (weight === 'extrabold' ? WEIGHTS.bold : WEIGHTS[weight]);
```

Comment:
coerce, is better

Reply:
perhaps, even create a separate weights hash object and just use it

Reply:
@iamstarkov I did that first, but then I went the function route because then we can change extrabold --> bold